platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -593,6 +593,18 @@ public float getTilt() {
         return nativeGetTilt(mapPointer);
     }
 
+    /**
+     * Run flight animation to change postion and zoom  of the map
+     * @param tilt Tilt angle in radians; 0 corresponds to straight down
+     * @param duration Time in milliseconds to ease to the given tilt
+     * @param ease Type of easing to use
+     */
+    public void flyTo(final double lon, final double lat, final float zoom, final float duration, final float speed) {
```

Comment:
I think we'll also want overloads for this, one with only duration and one with neither duration nor speed. It would be nice to specify only speed as well, but that won't work without named arguments.